src/Compilers/CSharp/Portable/Symbols/Synthesized/ReadOnlyListType/SynthesizedReadOnlyListIndexerProperty.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class SynthesizedReadOnlyListIndexerProperty : SynthesizedReadOnlyListPropertyBase
+    {
+        private readonly PropertySymbol _interfaceProperty;
+
+        internal SynthesizedReadOnlyListIndexerProperty(
+            SynthesizedReadOnlyListTypeSymbol containingType,
+            PropertySymbol interfaceProperty,
+            GenerateMethodBodyDelegate getAccessorBody,
+            GenerateMethodBodyDelegate? setAccessorBody) :
+            base(containingType)
```

Comment:
```suggestion
            GenerateMethodBodyDelegate? setAccessorBody)
            : base(containingType)
```

like with `?:` wrap before the token to keep constructs together and simplify reading.